src/input.h

```diff
@@ -634,6 +634,8 @@ class input_context
          */
         input_event get_raw_input();
 
+        std::tuple<point, bool> get_coordinates_text( const catacurses::window &capture_win ) const;
```

Comment:
```suggestion
        std::pair<point, bool> get_coordinates_text( const catacurses::window &capture_win ) const;
```

Reply:
Also why is it called `get_coordinates_text`? If I am getting it right it actually returns "text coordinates", so we better name it `get_text_coordinates`.